.circleci/config.yml

```diff
@@ -72,8 +72,8 @@ workflows:
                 - gemfiles/rails_6.1.gemfile
                 - gemfiles/rails_7.0.gemfile
                 - gemfiles/rails_7.1.gemfile
+                - gemfiles/rails_7.2.gemfile
               ruby_version:
```

Comment:
Can we use latest maintenance release for each version of Ruby?

Reply:
Should I also update each rails version in the Appraisals file to the latest maintenance release?

Reply:
That would be great. Thanks!